netilion/client.py

```diff
@@ -283,3 +283,29 @@ def get_asset_health_conditions(self, asset_id: int) -> list[AssetHealthConditio
     def get_asset_health_condition(self, health_condition_id: int) -> AssetHealthCondition:
         response = self.get(self.construct_url(self.ENDPOINT.ASSET_HEALTH_CONDITION, {"health_condition_id": health_condition_id}))
         return AssetHealthCondition.parse_from_api(response.json())
+
+    def post_asset_health_conditions(self, asset_id: int, health_conditions_id: list[int]) -> None:
+        url = self.construct_url(self.ENDPOINT.ASSET_HEALTH_CONDITIONS, {"asset_id": asset_id})
+        body = {
+            "health_conditions":
+                [{"id": health_condition_id} for health_condition_id in health_conditions_id]
+        }
+        response = self.post(url, json=body)
+        if response.status_code >= 300:
```

Comment:
As far as I understand the API will return 201 on a success. 
Maybe it would be better to raisse an exception if any other statuscode then 201 is received.
Similar in delete_asste_health_condition(...) there the API only mentions 204 for a success. 
So e.g. if reponse.status_code != 201 ...